package.json

```diff
@@ -1,18 +1,20 @@
 {
   "name": "resj",
+  "version": "0.3.0",
   "private": true,
   "dependencies": {
-    "@rails/activestorage": "^6.0.0",
-    "@rails/ujs": "^6.0.0",
-    "@rails/webpacker": "5.4.3",
     "choices.js": "^9.0.0",
     "mapbox-gl": "^2.6",
-    "stimulus": "^3.0",
-    "webpack": "^4.46.0",
-    "webpack-cli": "^3.3.12"
+    "stimulus": "^3.0"
   },
-  "version": "0.2.0",
   "devDependencies": {
+    "@babel/plugin-proposal-private-methods": "^7.16.0",
+    "@rails/activestorage": "^6.0.0",
+    "@rails/ujs": "^6.0.0",
+    "@rails/webpacker": "5.4.3",
+    "webpack": "^4.46.0",
+    "webpack-cli": "^4.3.1",
+    "@webpack-cli/serve": "^1.6.0",
```

Comment:
These are not development dependencies !

Reply:
True for `@rails/activestorage` and `@rails/ujs`. But the the others concern the bundling and such, which is not needed in the production app. Or am I missing something ?

Reply:
Looking at the default package.json in a Rails app, there are not under devDependencies. If we have another insight showing that it is safe to do so, go ahead.

Reply:
ok let's stick to common practice.